test/project_types/theme/commands/pull_test.rb

```diff
@@ -91,12 +92,15 @@ def test_pull_with_name
         @command.call([], "pull")
       end
 
-      def test_pull_with_root
+      def test_pull_with_empty_root
```

Comment:
I tried to test this with `shopify theme pull --path "…"`, as the `--path` parameter is also mentioned in https://shopify.dev/themes/tools/cli/commands#pull 
However, the CLI tells me that `--path` is not a valid option.
Are the docs on shopify.dev outdated?

Reply:
Haa, touché, I tested with CLI 2.x – my bad 👓